src/_includes/optional-types-2.0.html

```diff
@@ -0,0 +1,5 @@
+<aside class="alert alert-info" markdown="1">
+  **A note about Dart 2.0**:
+  As of Dart 2.0, types won't be optional. For more information, see
```

Comment:
The first sentence is misleading in the context of https://sz-www-1.firebaseapp.com/guides/language/effective-dart/design. We should either customize the text or come up with new text that works everywhere. Maybe:

As of Dart 2.0, types won't be optional, but you'll still be able to omit some type annotations.